inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/base/layer_test_utils.hpp

```diff
@@ -210,6 +215,8 @@ class LayerTestsCommon : public CommonTestUtils::TestsCommon {
 
 private:
     RefMode refMode = RefMode::INTERPRETER;
+    std::string GetTimestamp();
+    std::string GetTestName();
```

Comment:
1. `std::string GetTimestamp();` can be static
2. `std::string GetTestName();` can be const

Reply:
Sure, Added